airflow/scripts/git_scripts/aggregate_repo_activity.py

```diff
@@ -0,0 +1,85 @@
+
+import requests
+from collections import defaultdict
+from datetime import datetime, timedelta
+
+token = 'put_token_here'   # add your token
+headers = {'Authorization': f'token {token}', 'Accept': 'application/vnd.github.v3+json'}
+
+since = (datetime.now() - timedelta(days=7)).isoformat()
+until = datetime.now().isoformat()
+
```

Comment:
Create a class called "Github." The class will contain all the actions required to fetch information from Github.
By using a Class we can easily import it to other projects.

Reply:
Can I create a BaseClass: GitHub and SubClasses : GitHubUser and GitHubRepository  which will inherit from GitHub. 
The BaseClass will contain the fetch data function and the subclasses implement specific functionalities?

Reply:
Ofc.